src/Site/OAuth/OAuthBase.php

```diff
@@ -37,7 +38,10 @@ abstract protected function getOAuthProvider($redirectUri): AbstractProvider;
     public function oauthCallback(Request $request, Application $app)
     {
         $website = Website::get();
```

Comment:
Delete this line since it's not needed anymore.
```suggestion
```

Reply:
Done in latest commit.